source/commands.txt

```diff
@@ -0,0 +1,114 @@
+.. _vsce-commands:
+
+============================
+MongoDB for VS Code Commands
+============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This document lists the VS Code commands available for
+|vsce|. To open the Command Palette and execute these commands:
+
+.. include:: /includes/list-tables/command-palette-options.rst
+
+Connection Commands
+-------------------
+
+Use these commands to manage connections to your MongoDB deployments.
```

Comment:
Should we specify that we intend users to issue these *from* the command palette explicitly? VSCode unfortunately uses the same rough UI for command palette vs search, and you can open up the latter just by missing the shift key.